### PR TITLE
[MIRROR] Adds three brain traumas (feat. returning to monkey) 

### PR DESCRIPTION
**Pull request by NovaBot13**
Original PR: https://github.com/tgstation/tgstation/pull/82129
-----
## About The Pull Request

- Adds "Primal Instincts" brain trama. 
   - Special trauma
   - Randomly, your body will be taken over by monkey AI. 
   - You will still be in control of the body, but you will have to fight against the monkey ai for actions. 
   - There is a probability the monkey AI that takes control is aggressive, meaning it will attack anyone nearby. 
   - While primal, you can understand monkey language. 

- Adds "Kleptomania" brain trauma
   - Severe trauma
   - When you have empty hands, you will passively try to pick up things near you. 
   - There is no feedback message associated, so you may not even notice you did this. 
   - This effect is temporarily disabled if you have taken damage recently. 

- Adds "Possessive" brain trauma
   - Mild trauma
   - Randomly, your held items will become undroppable for a short to medium length of time. 

## Why It's Good For The Game

I was looking through AI stuff recently and remembered we support allowing AI controllers to work in cliented mobs, but we don't use it anywhere (outside of adminbus) 

So I wanted to add a brain trauma themed around that. Simple enough. 

But I didn't want to PR just a single trauma, that's boring so I thought of some additional ones. Just to spice up the other two trauma pools. I especially wanted to add some traumas that interact with inventory and items, because while a lot of our traumas involving how a person interacts with the world, not many involve how a person interacts with themself. 

## Changelog

:cl: Melbert
add: Adds 3 Traumas: Primal Instincts (special), Kleptomania (severe), and Possessive (mild)
/:cl:

### CODE REVIEW

**Comment by TheSS13Melon** [comment]
We should probably not have the first two as they might result in problems.

Third one's probably okay though.

**Comment by Iamgoofball** [comment]
primal instincts is fine because admin logs differentiate between forced interactions by the game and player choice interactions

File: code/datums/brain_damage/special.dm [hunk 1]
```diff
@@ -472,3 +472,55 @@
...
+		return
+
+	if(!SPT_PROB(3, seconds_per_tick))
+		return
+
+	owner.grant_language(/datum/language/monkey, UNDERSTOOD_LANGUAGE, TRAUMA_TRAIT)
+	owner.ai_controller.set_blackboard_key(BB_MONKEY_AGGRESSIVE, prob(75))
+	if(owner.ai_controller.ai_status == AI_STATUS_OFF)
+		owner.ai_controller.set_ai_status(AI_STATUS_ON)
+		owner.log_message("became controlled by monkey instincts ([owner.ai_controller.blackboard[BB_MONKEY_AGGRESSIVE] ? "aggressive" : "docile"])", LOG_ATTACK, color = "orange")
```
Comment (Iamgoofball) [inline code/datums/brain_damage/special.dm:518]: both this and kleptomania are fully logged so admins can tell what's up

File: code/datums/brain_damage/severe.dm [hunk 2]
```diff
@@ -431,3 +431,66 @@
...
+			continue
+		if(potential_stealable.anchored || !(potential_stealable.interaction_flags_item & INTERACT_ITEM_ATTACK_HAND_PICKUP))
+			continue
+		stealables += potential_stealable
+
+	for(var/obj/item/stealable as anything in shuffle(stealables))
+		if(!owner.CanReach(stealable, view_only = TRUE) || stealable.IsObscured())
+			continue
+		// Try to do a raw click on the item with one of our empty hands, to pick it up (duh)
+		owner.log_message("attempted to pick up (kleptomania)", LOG_ATTACK, color = "orange")
```
Comment (Iamgoofball) [inline code/datums/brain_damage/severe.dm:487]: see above

**Pull request merged**